### DISCUSSION TITLE
[Discussion] OTel distributions: Where to list them? Should we?

### DISCUSSION THREAD

**Discussion by theletterf** [open-telemetry/opentelemetry.io#3812] [Information Architecture]
An interesting question arose in #3774 concerning OpenTelemetry distributions, such as Honeycomb's, Splunk's, and so on: Should we list them in the Registry, perhaps creating a new category, put them in https://opentelemetry.io/ecosystem/vendors/, or do something else?

I'd go for the vendor route, creating a subpage just for distributions, or even pages for each distribution (that would have to be created and maintained by distribution owners). Each section or page would point to the repository and present the main differences or features of the distribution, ideally in a table. For Collector distributions we could even list the components, as they're currently metadata in YAML format.

**Comment by mhausenblas** [comment]
With my AWS Distro for OpenTelemetry (ADOT) product hat on: +1 to create a sub-page for distributions (one page per distro seems too much for starters).

**Comment by svrnm** [comment]
I tried to write a reply to this now for a few times, because I am conflicted if and how to **promote** distributions: 

On the one hand I think that by giving them a dedicated page, or by giving them more space on the vendors page (i.e. by updating the column with "Distribution: yes/no" to a link to the same), we give end-users (and especially vendors!) the feeling that a distribution is a **must have** and that without one a vendor is maybe worse than one that has a distribution. We experience this already when people add their vendor to the list, they keep the "checkbox checked" until we ask them to provide proof that their distro exists.

This goes against (my!) perception of distributions: having a distribution is a **neutral** thing, it is a design decision, that a vendor should make based on their specific requirements. Many will live happy without their own distribution, while others figure out that a distro for their platform/backend/pipeline/... is what they need.

On the other thand, I think it's a good thing to let people know that for specific cases there is a distribution that they can use to make their live easier. It can be the one click solution to get started with a specific vendor, or it can be the piece that makes your cloud services fly with otel. So by not mentioning them people might struggle with building their own collector, using vanilla SDKs, while a solution is around the corner.

To make a long story short: We should have them in our ecosystem pages, but we need to make sure that we communicate well enough that distributions are not simply a checkbox that makes your solution automatically better.

**Comment by theletterf** [comment]
I think the single page solution could work if we open it by stating what distributions are and that they're not necessary to fully harness the power of OpenTelemetry.

**Comment by svrnm** [comment]
> I think the single page solution could work if we open it by stating what distributions are and that they're not necessary to fully harness the power of OpenTelemetry.

Fair point.

**Comment by theletterf** [comment]
@svrnm Happy to give this one a try. It's fairly simple for me to find the Collector distros — less so with APM stuff. Where could I find data for an initial list?

**Comment by svrnm** [comment]
@theletterf good question, we have the honeycomb one in the registry, the lumigo in the PR, and a list of "yes" on the vendor page (@mhausenblas can point you to ADOT). I don't think we need to start with a perfect list, collect ~5 and make a draft how it may look like and we go from there

**Comment by theletterf** [comment]
Creating a PR for the single-page solution. Thanks all!

https://github.com/open-telemetry/opentelemetry.io/pull/3854
